src/pocketmine/item/Item.php

```diff
@@ -524,6 +542,27 @@ public function getEnchantment(int $id){
 
 		return null;
 	}
+	
+	/**
+	* @param int $id
+	* @param int $level
+	*/
+	public function removeEnchantment(int $id, int $level = null){
```

Comment:
You cannot add Sharpness 2 and Sharpness 1 to a weapon, only 1 of each enchantment is supported making the level irrelevant to remove enchantments.